Readme.md

```diff
@@ -167,6 +167,18 @@ Dapper.Contrib makes use of some optional attributes:
 * `[Write(true/false)]` -  this property is (not) writeable
 * `[Computed]` - this property is computed and should not be part of updates
 
+Custom parameter prefix
+-------
+By default Dapper.Contrib uses the character `@` as parameter prefix for SQL query and parameters collection. This can be changed throught the `GetParameterPrefixForQuery` and `GetParameterPrefixForParameterCollection` delegates:
```

Comment:
minor feedback: spell check on the word `throught`, is it supposed to be read as `through`?

Reply:
Done.